src/components/NavBar/NavBar.js

```diff
@@ -29,15 +34,17 @@ export const NavBar = () => {
         <li className="nav-action-item">
           <div className="badge-container">
             <i className="fas fa-shopping-cart fa-2x" aria-hidden="true"></i>
-            <span className="badge badge-icon">10</span>
-          </div>
-        </li>
-        <li className="nav-action-item">
-          <div className="badge-container">
-            <i className="far fa-heart fa-2x"></i>
-            <span className="badge badge-icon">10</span>
+            <span className="badge badge-icon">{userCart.length}</span>
           </div>
         </li>
+        <Link to="/wishlist">
+          <li className="nav-action-item">
+            <div className="badge-container">
+              <i className="far fa-heart fa-2x"></i>
+              <span className="badge badge-icon">{userWishlist.length}</span>
```

Comment:
suggestion : can hide the badge if wishlist/cart is empty (instead of showing 0)

Reply:
Hey! Thanks for suggesting this....I am planning on hiding these badges if user is not logged in.